.github/workflows/build-release.yml

```diff
@@ -0,0 +1,63 @@
+name: Build Release
+on:
+  push:
+    tags:
+    - v*
+jobs:
+  create-release:
+    runs-on: 'ubuntu-latest'
+    steps:
+    - name: Create Release
+      id: create-release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
+        draft: false
+        prerelease: false
```

Comment:
I usually label everything as `pre-release` initially to ensure there's some kind of "do not use this in production" warning. This allows some time to test the tarballs on various platforms before manually marking it as production-ready and publishing new versions of sharp that depend upon it.

The existing build script should probably do more verification of the resultant tarballs, which would then provide more confidence in their production readiness, but we're not quite there yet.

Reply:
I've changed the attribute to mark the GitHub Release as pre-release.

With regards to testing the tar-balls, we could create that as an additional job after the build and pass the artifacts between jobs.
Do you have a suggestion how to verify the tar-balls?

Reply:
Basic verification might be as simple as ensuring `ldd-cpp.so` has zero "not found" dependencies, at least for Linux. A more complex process could checkout sharp, force it to use dependencies from a given tarball and run all its unit tests. Let's merge these improvements first as they are independent of any future verification step.